data/Parvej2001.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Parvej Shaikh",
+  "displayStatsPublic" : true,
+  "type":"personal",
+  "bio": "Building Axstudios.tech and Skilled in Full Stack development.Exploring Cloud Native and Blockchain technology with a bachelor of engineering focused in Information Technology",
+  "avatar": "https://github.com/parvej2001.png",
```

Comment:
this is no longer needed:
```suggestion
```

Reply:
ok thankyou for letting me know. i will change it

Reply:
@SaraJaoude i have done the changes